erpnext/stock/doctype/stock_settings/stock_settings.json

```diff
@@ -19,6 +19,7 @@
   "auto_insert_price_list_rate_if_missing",
   "allow_negative_stock",
   "autoname_item",
+  "ignore_licenses_for_tax_calculation",
```

Comment:
I think the label should give more context can you rename to disable calculate cannabis taxes